docs/log_list_getter.rst

```diff
@@ -0,0 +1,40 @@
+Log List Getter
+==================
```

Comment:
The lines under headings need to be the same length as the headings (here and the two below).